pkg/reconciler/v1alpha1/route/resources/cluster_ingress.go

```diff
@@ -67,7 +65,7 @@ func MakeClusterIngress(r *servingv1alpha1.Route, tc *traffic.Config, ingressCla
 }
 
 func makeClusterIngressSpec(r *servingv1alpha1.Route, targets map[string]traffic.RevisionTargets) v1alpha1.IngressSpec {
-	// Domain should have been specified in route status
+	// Domain and should have been specified in route status
```

Comment:
and what?

Reply:
this was left over from an earlier iteration. Deleted and.